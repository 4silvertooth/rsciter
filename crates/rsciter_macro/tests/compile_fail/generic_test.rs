#[rsciter_macro::xmod]
mod with_generic {
    pub fn test<T: AsRef<str>>(_s: T) {
    }
}

fn main() {}