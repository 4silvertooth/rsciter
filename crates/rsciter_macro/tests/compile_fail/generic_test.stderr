error: #[rsciter::xmod] Generic functions are not supported!
 --> tests/compile_fail/generic_test.rs:3:16
  |
3 |     pub fn test<T: AsRef<str>>(_s: T) {
  |                ^
